azure-functions/acbs-function/src/functions/activity-update-facility-master.js

```diff
@@ -68,15 +44,37 @@ const handler = async (payload) => {
     }
 
     const { facilityId, acbsFacilityMasterInput, updateType, etag } = payload;
-
+    const mandatoryFields = [
```

Comment:
What's the reason for moving this inside the function?

Seems that all other similar ACBS functions have this defined at the top, I think this is easier to read/consume/maintain and should be aligned

Reply:
Good spot, have moved all `mandatoryFields` all inside function.
Reason for moving inside function is to ensure variables are declared in memory after passing payload validation.

Reply:
Not a blocker - maybe one for another PR.

Some instances of `mandatoryFields` take up a lot of space in the functions, maybe we could extract into some ACBS constants or something e.g

```ts
REQUIRED_FIELDS.FUNCTION_X = [...];
REQUIRED_FIELDS.FUNCTION_Y = [...];
REQUIRED_FIELDS.FUNCTION_Y = null;
```

Reply:
Sounds good.